src/main/java/nextstep/subway/ui/LineStationController.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineStationService;
+import nextstep.subway.dto.SectionRequest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/lines")
+public class LineStationController {
+    private final LineStationService lineStationService;
+
+    public LineStationController(LineStationService lineStationService) {
+        this.lineStationService = lineStationService;
+    }
+
+
+    @PostMapping("/{id}/sections")
+    public ResponseEntity addLineStation(@PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> addLineStation(@PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
```
제네릭형에 타입은 명시되는게 좋습니다!